cmd/kubeadm/app/util/version.go

```diff
@@ -62,6 +62,10 @@ var (
 //  latest-1    (latest release in 1.x, including alpha/beta)
 //  latest-1.0  (and similarly 1.1, 1.2, 1.3, ...)
 func KubernetesReleaseVersion(version string) (string, error) {
+	return releaseVersion(version, fetchFromURL)
+}
+
+func releaseVersion(version string, fetcher func(string, time.Duration) (string, error)) (string, error) {
```

Comment:
I'd rather want the name to contain the word "kubernetes" or "k8s" somewhere. This is not a local func for the file, but local for the whole util package, hence it needs more context.